### ISSUE TITLE
Add direct access to accessibility label for background elements

### ISSUE THREAD

**Issue by barklund** [ampproject/amp-wp#3185]
_Spawned from #2983_

As a user I want to be able to see and edit the accessibility label for background elements directly in the block editor for a given story page.

The value should be prepopulated from the media editor:
- For background images, the field should be pre-populated with the value from the `alt` attribute from the media editor.
- For background videos, the field should be pre-populated with the value from the `title` attribute from the media editor.

The field should be a simple single-line text field placed near the _Change Media_ button for the background block element in the _Background Media_ section with a descriptive label.

The value should be saved with the individual page with the given media element and not on the media element itself. If multiple pages/stories use the same media element, they can have different accessibility labels.

The value should be rendered as part of the AMP story output as follows:
- For background images, the field should be rendered as the `alt` attribute on the `<amp-img>` node.
- For background images, the field should be rendered as the `aria-label` attribute on the `<amp-video>` node.

**Comment by csossi** [comment]
Verified in QA

image - "src="https://story-test-wordpress-amp.pantheonsite.io/wp-content/uploads/2019/08/IMG_20180809_175040.jpg" alt="Alt Text Goes Here" "

video - "<amp-video layout="fill" aria-label="Alt Text Goes Here" src="https://story-test-wordpress-amp.pantheonsite.io/wp-content/uploads/2019/08/Pexels-Videos-1194714-6.mp4" "

**Comment by swissspidy** [comment]
Screencast: https://cloudup.com/cUf1gCjwwEq
